src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -19,7 +19,7 @@ public class AddCommandParser implements Parser<AddCommand<?>> {
     public AddCommand<?> parse(String args) throws ParseException {
 
         String entity = args.trim().split(" ")[0];
-        String addArgs = args.replace(" " + entity, "");
+        String addArgs = args.replace(entity, "");
 
         switch (entity) {
```

Comment:
This can only be done after keng hian's PR merges

Reply:
After trying it out, I realise that I cannot use Keng Hian's parser function for this as the error message says company, contact, job, or all which is technically not correct since edit and find only have contact. Additionally, if the first arg in for this is not the entity, it will say invalid entity "NEXT_ARG" which might not be the best representation as compared to the current implementation which displays the full message usage.